src/main/java/com/powsybl/openloadflow/OpenLoadFlowProvider.java

```diff
@@ -151,7 +151,7 @@ private LoadFlowResult runAc(Network network, LoadFlowParameters parameters, Rep
                                                                             result.getNetwork().getNumSC(),
                                                                             status,
                                                                             result.getNewtonRaphsonIterations(),
-                                                                            slackBusId,
+                                                                            slackBusId, // FIXME manage multiple slack buses
```

Comment:
Are you going to fix this in that PR?

Reply:
That would require to make some change in the loadflow API so not in this PR.